static/app/components/organizations/projectSelector/index.tsx

```diff
@@ -157,9 +160,14 @@ const ProjectSelector = ({
     ];
   };
 
+  const handlePinClick = () => {
+    PageFiltersActions.pin('projects', !pinned);
```

Comment:
Trigger this through the action creator actually https://github.com/getsentry/sentry/blob/f3467f21acad7898448a01462730e969dbf1a5d1/static/app/actionCreators/pageFilters.tsx#L293

Reply:
ahh yup makes sense with your other PR, the actionCreator also handles the persistence